jobs/route_emitter_windows/templates/route_emitter.json.erb

```diff
@@ -78,6 +78,7 @@
     nats_client_key_file: "#{conf_dir}/certs/nats/client.key",
     communication_timeout: "30s",
     sync_interval: "#{p("diego.route_emitter.sync_interval_in_seconds")}s",
+    jitter_factor: p("diego.route_emitter.jitter_factor"),
```

Comment:
please check the value first and throw an alert if the value is illegal, e.g.

- must be float
- can't be < 0.0
- can't be > 1.0